comps/lvms/README.md

```diff
@@ -1,111 +1,18 @@
-# LVM Microservice
+# Language Vision Model (LVM) Microservice
 
-Visual Question and Answering is one of the multimodal tasks empowered by LVMs (Large Visual Models). This microservice supports visual Q&A by using LLaVA as the base large visual model. It accepts two inputs: a prompt and an image. It outputs the answer to the prompt about the image.
+This microservice, designed for Large Vision Model (LVM) Inference, processes input consisting of a query string and/or image prompts. It constructs a prompt based on the input, which is then used to perform inference with a large vision model (e.g., LLaVA). The service delivers the inference results as output.
 
-# 🚀1. Start Microservice with Python (Option 1)
+A prerequisite for using this microservice is that users must have an LVM service (transformers or Prediction Guard) already running. Overall, this microservice offers a streamlined way to integrate large vision model inference into applications, requiring minimal setup from the user. This allows for the seamless processing of text and image prompts to generate intelligent, context-aware responses.
 
-## 1.1 Install Requirements
+## Getting started with transformers + fastAPI services
 
-```bash
-pip install -r requirements.txt
-```
+The [transformers](transformers) directory contains instructions for running services that serve predictions from a LLaVA LVM. Two services must be spun up to run the LVM:
```

Comment:
I think you should also delete all the content under the legacy folder https://github.com/predictionguard/pg-GenAIComps/tree/main/comps/lvms/llava, since they are moved to `transformers`. Let me know if I miss something!

Reply:
Ah, good catch. Yes, that was the intention, and I've deleted the unnecessary files.